lib/datasource/docker/index.ts

```diff
@@ -328,6 +328,52 @@ async function getManifestResponse(
   }
 }
 
+async function getConfigDigest(
+  registry: string,
+  repository: string,
+  tag: string
+): Promise<string> {
+  const manifestResponse = await getManifestResponse(registry, repository, tag);
+  // If getting the manifest fails here, then abort
+  // This means that the latest tag doesn't have a manifest, which shouldn't
+  // be possible
+  // istanbul ignore if
+  if (!manifestResponse) {
+    logger.debug(
+      {
+        registry,
+        dockerRepository: repository,
+        tag,
+      },
+      'docker registry failure: failed to get manifest for tag'
+    );
```

Comment:
I think we can remove this, as `getManifestResponse` already logs any errors before returning `null`
```suggestion
```